src/app/zap-templates/zcl/data-model/chip/application-launcher-cluster.xml

```diff
@@ -26,57 +26,45 @@ limitations under the License.
     <description>This cluster provides an interface for launching content on a media player device such as a TV or Speaker.</description>
 
     <attribute side="server" code="0x0000" define="APPLICATION_LAUNCHER_LIST" type="ARRAY" entryType="INT16U" length="254" writable="false" optional="false">application launcher list</attribute>
-    <attribute side="server" code="0x0001" define="APPLICATION_LAUNCHER_CURRENT_APP" type="ApplicationLauncherEndpoint" writable="true" optional="false">application launcher app</attribute>
+    <attribute side="server" code="0x0001" define="APPLICATION_LAUNCHER_CURRENT_APP" type="ApplicationEP" writable="true" optional="false">application launcher app</attribute>
 
-    <command source="client" code="0x00" name="LaunchApp" optional="false">
+    <command source="client" code="0x00" name="LaunchAppRequest" response="LauncherResponse" optional="false">
```

Comment:
Please file a spec issue on the inconsistency between "Launcher" and "Launcher Response" in various places.

Reply:
Added: https://github.com/CHIP-Specifications/connectedhomeip-spec/issues/4798

I cannot assing label nor assignee, can you help?